ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn.yml

```diff
@@ -207,6 +207,13 @@ router_bgp:
         send: any
     layer_2_fec_in_place_update:
       enabled: true
+    evpn_ethernet_segment:
+      - domain: remote
+        identifier: '0022:2222:2222:2222:2222'
+        route_target_import: '22:22:22:22:22:22'
+      - domain: local
+        identifier: '0011:1111:1111:1111:1111'
+        route_target_import: '11:11:11:11:11:11'
```

Comment:
you don't need to put the test in both `router-bgp-evpn` and `router-bgp-evpn-mpls`